fire/fire_import_test.py

```diff
@@ -12,16 +12,19 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-import fire
-
+import mock
+import sys
 import unittest
 
+import fire
+
 
 class FireImportTest(unittest.TestCase):
   """Tests importing Fire."""
 
   def testFire(self):
-    fire.Fire()
+    with mock.patch.object(sys, 'argv', ['commandname']):
+        fire.Fire()
```

Comment:
whoops - translating between different style guides - do you have a set of lint settings that you want to apply to the repo? (or is there some google standard that can be added to travis runs)